docs/content/2.deploy/providers/netlify.md

```diff
@@ -35,3 +35,25 @@ Nitro output can directly run the server at the edge. Closer to your users.
 
 On-demand Builders are serverless functions used to generate web content as needed that’s automatically cached on Netlify’s Edge CDN. They enable you to build pages for your site when a user visits them for the first time and then cache them at the edge for subsequent visits.  ([Read More](https://docs.netlify.com/configure-builds/on-demand-builders/))
 
+## Handling atomic deploys
+
+Netlify uses atomic deploys, which means that all old chunk files will become invalid when a deploy occurs.
+This causes errors when existing clients try to load chunk files which no longer exists.
+
+Nuxt, for example, has default behaviour to reload the page in some circumstances, but this is not always
+desirable and it is possible to do better by serving Netlify's permanent link to each deploy, which remains valid.
+
+For example in Nuxt, add this to `nuxt.config.ts`
+
+    $production: {
+        app: {
+            cdnURL: '/netlify/' + process.env.DEPLOY_URL?.replace('https://', ''),
+        },
+    },
+
+Then in the `_redirects` file which Netlify will pick up, proxy it:
+
+    /netlify/* https://:splat 200!
+
+Proxying is better than serving the files directly because some security software (e.g. Streamline3)
+will block scripts served from a different domain.
```

Comment:
A short & concrete example might be helpful. E.g.:

> For example, if you might be using the custom domain `https://example.com/` and the Netlify account `example.netlify.app`. In this case, `process.env.DEPLOY_URL` might be something like `https://5b243e66dd6a547b4fee73ae--example.netlify.app`.
> 
> Setting the `cdnURL` as above will cause Nuxt to try loading scripts from `https://example.com/netlify/5b243e66dd6a547b4fee73ae--example.netlify.app`, a non-existent URL within the custom domain. [Netlify will proxy](https://docs.netlify.com/routing/redirects/rewrites-proxies/) this back to the original `DEPLOY_URL` address without the HTTP client being aware of the domain change.

Reply:
I was a bit wary of swamping the page with just this topic, but since you've suggested it, I've added in that in.  Thanks.

Reply:
Yeah, I was just following along from https://github.com/nuxt/nuxt/issues/20950 and lack context on these docs. I won't be offended if it's too much info for this page!